content/getting-started-drf-angularjs-part-1.md

```diff
@@ -0,0 +1,150 @@
+
+## Goals of this Section
+
+This section focuses on Django, covering the following topics:
+
+* [Brief introduction to Django, Django REST Framework, and AngularJS](#introduction)
+* [Initial project setup and creation](#project-creation)
+* [Folder structure modification to support both front- and back-end development](#folder-structure)
+
+<a name="introduction"></a>
+## A Brief Introduction: Django, DRF and AngularJS
+
+RESTful APIs have become increasingly popular among modern web applications since they provide a standard means to interact with resources across applications.  Conforming to RESTful constraints can create web applications that are both high-performing and maintainable.  The Django and DRF frameworks provide developers with fast and secure ways to create RESTful web applications packed full of useful features, such as an extensive ORM, serialization, custom authentication and permissions classes, and browsable APIs among others.
+
+Think of Django as the database manager for your server.  The Django ORM provides powerful ways to locally setup and manage database tables and the data within them.  DRF is the external window into your database.  DRF provides the means to create powerful APIs to access application data while allowing developers to customize permissions, authentication, filtering, and more.  AngularJS if a front-end framework that provides two-way data binding between HTML and Javascript to dynamically display data.
+
+At TrackMaven, we use Django and DRF as the main backend frameworks for our web application development and AngularJS for front-end development.  Through these, we find that web application development is both straight-forward and extensible enough to fit into our growing demands while maintaining RESTful principles.
+
+<a name="project-creation"></a>
+## Setting up a New Project
+### Creating an Initial Project
+
+Getting started with Django and DRF is quite easy.  Django provides a startup script that builds a starter project with default configurations, an initial `/admin/` API endpoint, and a management command file for running the application.
+
+To start, let's create a virtual environment for our sample project and download our required packages.
+
+```shell
+$ mkvirtualenv drf-sample
+$ pip install django==1.8.5
+$ pip install djangorestframework==3.3.0
+```
+
+Within the virtual environment, we can run the Django start-up script to create a new project named `drf_sample`
+
+```
+$ django-admin startproject drf_sample
+```
+
+We now have a new project folder named `drf_sample` with the following structure:
+```
+drf_sample/
+├── drf_sample
+│   ├── __init__.py
+│   ├── settings.py
+│   ├── urls.py
+│   └── wsgi.py
+└── manage.py
+```
+Cool, but what do these files actually do?
+
+<dl>
+  <dt>settings.py</dt>
```

Comment:
If you want the markdown `<backtick>` to happen here in HTML, you'll need to surround `settings.py` with `<code>`:

```
<code>settings.py</code>
```

Only add that if you want to...